hudi-tests-common/pom.xml

```diff
@@ -227,5 +227,13 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
Does `hudi-utilities-bundle` need to include these dependencies?  Does the same code work in `spark-submit` or `spark-shell` with `hudi-utilities-bundle` (is it only a test dependency issue or bundle issue)?

Reply:
I haven't figured it out. I will remove the dependency from  hoodie-tests-common to confirm.

Reply:
Sounds like the dependency is needed not only for testing.